rita_exit/src/database/mod.rs

```diff
@@ -517,3 +526,88 @@ pub fn enforce_exit_clients(client_data: &mut RitaExitData) -> Result<(), Box<Ri
     client_data.set_debt_actions(new_debt_actions);
     Ok(())
 }
+
+/// Removes client ip assignments for clients that have not checked in for 1 hour
+pub fn teardown_inactive_clients(client_data: &mut RitaExitData) {
+    let inactive_list = client_data.get_inactive_list();
+    // check for snat mode
+    let ipv4_mode = client_data.get_ipv4_nat_mode();
+    if let ExitIpv4RoutingSettings::SNAT { .. } = ipv4_mode {
```

Comment:
you can check for the mode before getting the inactive list and save some commands/operations